styles.css

```diff
@@ -338,8 +498,765 @@ footer {
+  }
+
+  #language-template {
+    grid-row: 1;
+    grid-column: 1;
+    width: 20.1vw;
+    height: 20.1vw;
+    background-color: #1c528a;
+    display: flex;
+    flex-direction: column;
+    margin: 0 0 0 0;
+  }
+
+  #frameworks-template {
+    grid-row: 1;
+    grid-column: 2;
+    width: 20.1vw;
+    height: 20.1vw;
+    background-color: #e2505c;
+    display: flex;
+    flex-direction: column;
+    margin: 0 0 0 0;
+  }
+
+  #skills-template {
+    grid-row: 2;
+    grid-column: 1;
+    width: 20.1vw;
+    height: 20.1vw;
+    background-color: #febf52;
+    display: flex;
+    flex-direction: column;
+    margin: 0 0 0 0;
+  }
+
+  #dot-grid-template {
+    display: block;
+    grid-row: 2;
+    grid-column: 2;
+    background-image: url('Images SVG/Bluesquare.svg');
+    background-repeat: no-repeat;
+    background-size: 4.3vw;
+    background-position: right top;
+    margin-left: 4.7vw;
+    padding-top: 4vw;
+    margin-top: 5vw;
+  }
+
+  .templates-title {
+    width: 8vw;
+    height: 3vw;
+    margin-top: 3vw;
+    margin-left: 5.7vw;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 1.29vw;
+    line-height: 1.83vw;
+    color: white;
+  }
+
+  .list-container {
+    width: 11vw;
+    height: 8vw;
+    display: flex;
+    flex-direction: column;
+    list-style-type: disc;
+    color: white;
+    margin: 1.5vw 0 0 7vw;
+  }
+
+  .template-item {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 1vw;
+    line-height: 1.7vw;
+    letter-spacing: -0.0125em;
+    color: white;
+  }
+
+  .headlineabout {
+    margin-block: 3vw;
+    width: 40vw;
+    height: 40vw;
+    display: flex;
+    flex-direction: column;
+    align-content: left;
+  }
+
+  .titleabout {
+    font-family: 'Playfair Display', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 7.7vw;
+    line-height: 100%;
+    color: #1a2236;
+    padding-bottom: 1vw;
+  }
+
+  .mynameintroabout {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 0.95vw;
+    line-height: 1.6vw;
+    letter-spacing: -0.01em;
+    color: #1a2236;
+    padding-bottom: 1vw;
+  }
+
+  .buttonboxabout {
+    width: 16.5vw;
+    height: 4vw;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.2vw;
+    line-height: 4vw;
+    color: #fff;
+    cursor: pointer;
+    background: #1a2236;
+  }
+
+  #downloadimageabout {
+    margin-inline: 1.2vw;
+    margin-top: 0.3vw;
+  }
+
+  .contact {
+    background-image: url('Images SVG/image-geometry_6.svg'), url('Images SVG/image-geometry_7.svg');
+    background-position: left top, right bottom;
+    background-repeat: no-repeat;
+    margin-left: 0;
+    margin-top: 0;
+    padding-top: 0;
+    width: 100vw;
+    height: 62vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
```

Comment:
- [ ] As requested by the previous reviewer, please adjust the spacing around the background icons and center the form.
![image](https://user-images.githubusercontent.com/91486791/230727559-e74d2b01-8ea2-4445-a5de-9950efe47180.png)